sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/NUnitTestResultsFileParser.java

```diff
@@ -32,6 +32,7 @@
 
   @Override
   public void accept(File file, UnitTestResults unitTestResults) {
+    LOG.debug("The current user dir is '{}'.", System.getProperty("user.dir"));
```

Comment:
Nothing to log here? `NCover3ReportParser.java` reported it's progress on the fly inside the document. At least detect and log presence "test-results"+"test run" and "no test-results found" and "no test-run".

Reply:
good point. I didn't really go into UTs as I was focusing on the coverage import, but this is a low hanging fruit